src/plugins/compare/file_header/code/file_header.py

```diff
@@ -1,3 +1,5 @@
+from __future__ import annotations
```

Comment:
This is needed to support [PEP 585](https://peps.python.org/pep-0585/) and [PEP 604](https://peps.python.org/pep-0604/).  The reason for this is, that normally you would need at least Python 3.10 to support this syntax and since FACT still supports Python 3.8, we need this future import to add backwards compatibility (see also [PEP 563](https://peps.python.org/pep-0563/)).